services/core-api/app/api/incidents/response_models.py

```diff
@@ -83,7 +87,9 @@
         'mine_incident_no': fields.String,
         'documents': fields.List(fields.Nested(MINE_INCIDENT_DOCUMENT_MODEL)),
         'recommendations': fields.List(fields.Nested(
-            MINE_INCIDENT_RECOMMENDATION_MODEL))
+            MINE_INCIDENT_RECOMMENDATION_MODEL)),
+        'mine_incident_category_xref': fields.List(fields.Nested(
```

Comment:
this just refers directly to the many-to-many relationship and reuse the existing MINE_INCIDENT_DOCUMENT_MODEL.

Reply:
I'm not sure what you mean exactly.

Reply:
change line 91/92 to 

`'<relationship_name>' : fields.List(fields.Nested( MINE_INCIDENT_DOCUMENT_MODEL))`

Reply:
Document?

Reply:
Category, sorry.